gatsby-config.js

```diff
@@ -51,6 +51,10 @@ module.exports = {
             title: "Introduction to Journaling",
             path: "guides/journaling_intro.md",
           },
+          {
+            title: "App Builder Auto Subscriptions",
```

Comment:
We have been consistent in our usage of the term "event registration" for I/O Events. I think we should not introduce another term `subscription` for this. 

How about `Automatic event registrations using App Builder`? Or do you have any other suggestion?

Reply:
This is not a part of app builder. I would like to drop `app builder` from the phrase to avoid confusion

Reply:
fixed

Reply:
Thanks! Can we please remove the `subscriptions` word from the content of `autosubscriptions.md` too? And also rename the file accordingly?

Reply:
yes, done